lib/sql-string.js

```diff
@@ -35,6 +35,11 @@ function escape(val, timeZone, dialect, format) {
     break;
   }
 
+  // moment
+  if (val.toDate) {
```

Comment:
We need specific moment support here, I would rather avoid duck typing. You could also use `instanceof` instead of `isMoment`. But we needs to be sure its moment nothing else.